### PR TITLE
fix: Send packages array in sdk info

### PR DESCRIPTION
**Pull request by untitaker**
Depends on https://github.com/getsentry/rust-sentry-types/pull/24

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -34,7 +34,7 @@ backtrace = { version = "0.3.8", optional = true }
 url = { version = "1.7.0", optional = true }
 failure = { version = "0.1.2", optional = true }
 log = { version = "0.4.3", optional = true, features = ["std"] }
-sentry-types = "0.5.3"
+sentry-types = "0.6.3"
```
Comment (untitaker) [inline Cargo.toml]: Version does not exist yet

**Comment by untitaker** [comment]
Yeah. Not sure what to do about actix. It's also not part of the integration array.

**Comment by untitaker** [comment]
@mitsuhiko clue what to do about actix integration?

**Comment by mitsuhiko** [comment]
@untitaker in the event handler we just add it to the packages array

**Comment by untitaker** [comment]
ok. Shouldn't we do the same for `integrations`?

**Comment by mitsuhiko** [comment]
@untitaker please verify with @HazAT if that list is still to be populated.

**Comment by jan-auer** [comment]
@untitaker can we revisit this, please? `master` is now updated and has new `sentry-types`.

**Comment by jan-auer** [comment]
nvm, this is obsolete now as everything has been implemented as part of #82

**Pull request closed without merging**